docs/partials/sig-link-parent.hbs

```diff
@@ -0,0 +1,13 @@
+{{#if name}}{{#sig~}}
```

Comment:
What's this file for? Please explain it(and better to document it).

Reply:
This file is the jsdoc2md official partials: https://github.com/jsdoc2md/dmd/blob/master/partials/shared/signature/sig-link-parent.hbs

I  add link `chatie.io/wechaty` here to custom the linkable toc